### PR TITLE
Week 43 Dependency Updates

### PR DESCRIPTION
**Pull request by TheBitShepherd**
* Nothing for release notes

## Changes
Noticed an unused dependency `myst` that was causing some conflicts, so it has been removed 🔪  .

### CODE REVIEW

File: requirements/dev.in [hunk 1]
```diff
@@ -7,7 +7,6 @@ flake8<4
 furo
 isort
 jsonschema
-myst
```
Comment (TheBitShepherd) [inline requirements/dev.in:10]: 🔪

Comment (jstvz) [reply requirements/dev.in:10]: 💀

File: requirements/dev.txt [hunk 2]
```diff
@@ -229,44 +193,32 @@ requests==2.28.1
...
 rich==12.6.0
     # via
     #   -c requirements/prod.txt
     #   twine
-rsa==4.9
-    # via google-auth
-secretstorage==3.3.3
-    # via
-    #   -c requirements/prod.txt
-    #   keyring
```
Comment (jstvz) [inline requirements/dev.txt:255]: Required on Linux (Heroku), but let the web apps sort it out.

**Review by jstvz — APPROVED** [review]
Nice, dropping a few deps 😏

**Pull request merged**